src/GuiBase/Utils/KeyMappingManager.cpp

```diff
@@ -4,6 +4,15 @@
 
 namespace Ra {
 namespace Gui {
+
+//char const* const KeyMappingManager::KeyMappingActionNames[] = {
+const std::string KeyMappingManager::KeyMappingActionNames[] = {
```

Comment:
This is redundant with the services offered by ``Q_ENUM( KeyMappingAction )`` (line 56 of KeyMappingManager.hpp) and the operation ``m_metaEnumAction.valueToKey(action)`` that convert an enum value of type ``KeyMappingAction ``  to a ``const char *`` (see Qt documentation). The ``QMetaEnum m_metaEnumAction`` is required for parsing the xml configuration file. Nevertheless, we can keep this as its usage is Qt independent and do not need to have access to the required ``QMetaEnum`` and  the``friend std::ostream & operator <<(std::ostream& out, KeyMappingManager::KeyMappingAction action)`` operator could be used everywhere.

Reply:
What about this?

Reply:
We will keep both solutions for now.